docs/view_components/energy_system_component.rst

```diff
@@ -31,29 +30,31 @@ Attributes
 
     Requirements index:
 
-**Inward energy connection**
+**Input energy flow**
     A list of other components's ids which are connected to this component and provide it with energy
 
     Requirements index: 1, 5, 6
 
-**Outward energy connection**
+**Output energy flow**
     A list of other components's ids which are connected to this component and drain energy from it
 
     Requirements index: 1, 5, 6
 
 Actions
 ^^^^^^^
 
-1. Double-clicking on the rendering of the energy system component allow the user to edit its attributes (except id)
+1. Double-clicking on the rendering of the *energy system component* allow the user to edit its attributes (except id)
+2. As long as the *energy system component* is not connected correctly to another *energy system component* the Icon-frame is displayed in another colour
```

Comment:
I would say this is a requirement, as there is nothing "actionnable" on the user side here